GNUmakefile

```diff
@@ -67,7 +67,6 @@ abigen: ## Build & install abigen.
 
 .PHONY: go-solidity-wrappers
 go-solidity-wrappers: pnpmdep abigen ## Recompiles solidity contracts and their go wrappers.
-	./contracts/scripts/native_solc_compile_all
```

Comment:
This seems important, no?

Reply:
this script is run first thing here 
`https://github.com/smartcontractkit/chainlink/blob/develop/core/gethwrappers/go_generate.go#L6`
this line was just a duplicate

Reply:
Hmmm are you sure? I see `./generation/compile_contracts.sh` at that link, and the failing solidity tests run `make go-solidity-wrappers` directly: https://github.com/smartcontractkit/chainlink/blob/f2b10ae4d6e91159692e2a0e3757a2f694dc2d9c/.github/workflows/solidity.yml#L218
https://github.com/smartcontractkit/chainlink/actions/runs/4495358702/jobs/7908898697#step:7:1

Reply:
https://github.com/smartcontractkit/chainlink/blob/develop/core/gethwrappers/generation/compile_contracts.sh#L6